modflow_devtools/netcdf.py

```diff
@@ -0,0 +1,284 @@
+from dataclasses import dataclass, field
+from typing import Any
+
+import numpy as np
+import xarray as xr
+
+from modflow_devtools.netcdf_schema import NetCDFModel, get_dfn, validate
+
+DNODATA = np.float64(3.0e30)  # MF6 DNODATA constant
+FILLNA_INT32 = np.int32(-2147483647)  # netcdf-fortran NF90_FILL_INT
+FILLNA_INT64 = np.int64(-2147483647)
+FILLNA_FLOAT64 = np.float64(9.96920996838687e36)  # netcdf-fortran NF90_FILL_DOUBLE
+
+
+@dataclass
+class NetCDFPackageCfg:
```

Comment:
Suggest consistent naming of the user-facing model and package config objects, maybe dropping "Cfg" and "Input" and just calling them `NetCDFModel` and `NetCDFPackage`?

Just to mention for passersby, we discussed offline it would be nice if there were a single object model whose structure matches the schema and which exposes a template construction API via e.g. classmethods, instead of having two parallel object models, one schema, one user-facing. But I know you were still looking at the feasibility of that and it may come later